index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
```

Comment:
Please include the Linters Validation and the required structure:
- include the language in the HTML tag.
- include the Meta tags inside the Head Tag.
- include The Title tag inside the Head Tag.